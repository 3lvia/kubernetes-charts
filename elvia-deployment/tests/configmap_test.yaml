suite: test configmap
templates:
  - configmap.yaml
tests:
  - it: configmap snapshot should match
    set:
      image.tag: latest
      name: demo-app
      namespace: core
      microserviceType: worker
      environment: dev
      replicaCount: 1
      resources:
        requests:
          cpu: 100m
          memory: 128Mi
        limits:
          cpu: 400m
          memory: 500Mi
      configmap:
        dev:
          testmode: "true"
          elvid.authority: https://elvid.test-elvia.io/
        test:
          testmode: "false"
          elvid.authority: https://elvid.test-elvia.io/
        prod:
          testmode: "false"
          elvid.authority: https://elvid.elvia.io/
    asserts:
      - matchSnapshot: {}
