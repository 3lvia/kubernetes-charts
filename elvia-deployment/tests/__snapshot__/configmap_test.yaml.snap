configmap snapshot should match:
  1: |
    apiVersion: v1
    data:
      elvid.authority: https://elvid.test-elvia.io/
      testmode: "true"
    kind: ConfigMap
    metadata:
      name: demo-app
      namespace: core
